docs/src/examples/components/Dropdown/Rtl/DropdownExample.rtl.steps.ts

```diff
@@ -1,17 +1,17 @@
 import { Dropdown } from '@stardust-ui/react'
+import { applyCommonThemesBeforeStep } from '../../../screenerStepsUtils'
 
 const selectors = {
   triggerButton: `.${Dropdown.slotClassNames.triggerButton}`,
   item: (itemIndex: number) => `.${Dropdown.slotClassNames.itemsList} li:nth-child(${itemIndex})`,
 }
 
-const steps: ScreenerSteps = [
-  steps =>
-    steps
-      .click(selectors.triggerButton)
-      .snapshot('RTL: Shows list')
-      .click(selectors.item(3))
-      .snapshot('RTL: Selects an item'),
-]
+const steps: ScreenerSteps = applyCommonThemesBeforeStep(steps =>
```

Comment:
this is the current approach for testing `DropdownExample.shorthand.rtl.tsx` screenshot in all themes, we plan to simplify it, see ISSUES in Description

Reply:
fixed, we have better approach now